ipaserver/plugins/dogtag.py

```diff
@@ -1238,29 +1238,16 @@ def _parse_dogtag_error(body):
 
     def __init__(self, api):
         if api.env.in_tree:
-            self.sec_dir = api.env.dot_ipa + os.sep + 'alias'
-            self.pwd_file = self.sec_dir + os.sep + '.pwd'
+            self.ca_cert = os.path.join(api.env.dot_ipa, 'ca.crt')
         else:
-            self.sec_dir = paths.HTTPD_ALIAS_DIR
-            self.pwd_file = paths.ALIAS_PWDFILE_TXT
-        self.noise_file = self.sec_dir + os.sep + '.noise'
-        self.ipa_key_size = "2048"
-        self.ipa_certificate_nickname = "ipaCert"
-        self.ca_certificate_nickname = "caCert"
-        self._read_password()
+            self.ca_cert = paths.IPA_CA_CRT
+        self.client_certfile = paths.RA_AGENT_PEM
```

Comment:
Note to myself: RA_AGENT_PEM will need to move to `dot_ipa` as well. Testing guide on Wiki will need updating, too.